### DISCUSSION TITLE
please help!.. i can't gift stuff to Monika

### DISCUSSION THREAD

**Discussion by dontworryabouttheusername** [Monika-After-Story/MonikaModDev#9936] [QA - tech support + general]
I recently did a factory rese ont my pc..
downloaded a fresh copy of ddlc, downloaded the mod and loaded the backup persistent files..
I noticed a letter that says i can add a deck of cards "noudeck.gift".. but when i did, it doesn't have the usual "this will change the file"
instead it just turns into  a regular .txt with a name "noudeck.gift" instead of turning into a gift file.

**Comment by dontworryabouttheusername** [comment]
i tried with the usual coffee as well.. but that too just turns into a .txt as well
![Screenshot (1)](https://user-images.githubusercontent.com/54071609/224383915-244a2014-b429-4d29-ab01-e14100d573d8.png)

**Comment by Booplicate** [comment]
Enable this setting
![image](https://user-images.githubusercontent.com/53382877/224383962-abf4463e-f03e-4d74-9b31-f08b7c44588a.png)

**Reply by dontworryabouttheusername** [reply]
ohh sorry never saw this.. im sorry kinda new to github, but thanks i already resolved this issue
